srcpkgs/ocs-url/patches/prepare.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/scripts/prepare b/scripts/prepare
+index 0186aff..12e397a 100755
+--- a/scripts/prepare
++++ b/scripts/prepare
+@@ -3,5 +3,5 @@
+ PROJDIR="$(cd "$(dirname "${0}")/../" && pwd)"
+ 
+ if [[ ! -d "${PROJDIR}/lib/qtil" ]]; then
+-    git clone https://github.com/akiraohgaki/qtil.git -b release-0.4.0 --single-branch --depth=1 "${PROJDIR}/lib/qtil"
++    git clone https://github.com/akiraohgaki/qtil.git -b v0.4.0 --single-branch --depth=1 "${PROJDIR}/lib/qtil"
+ fi
```

Comment:
Does this need to be changed? You should directly download this repository in distfiles and move it into the correct place, because `git` shouldn't even be run.